setup.cfg

```diff
@@ -20,22 +20,17 @@ classifiers =
   Topic :: Software Development
   Topic :: Scientific/Engineering
   Operating System :: Microsoft :: Windows
-  Operating System :: POSIX
-  Operating System :: Unix
+  Operating System :: POSIX :: Linux
   Operating System :: MacOS
   Programming Language :: Python
-  Programming Language :: Python :: 3.7
-  Programming Language :: Python :: 3.8
-  Programming Language :: Python :: 3.9
 project_urls =
   Documentation = https://mne.tools/mne-bids
   Bug Reports = https://github.com/mne-tools/mne-bids/issues
   Source = https://github.com/mne-tools/mne-bids
 
 [options]
-python_requires = ~= 3.7
```

Comment:
why remove this?

Reply:
It's implicitly defined through our MNE-Python dependency. I thought we could avoid this redundancy, but can revert if you want to have this

Reply:
I'd like to keep it, please :)